app/assets/stylesheets/mortgage_calculator/components/_stamp-duty-calc.scss

```diff
@@ -1,3 +1,13 @@
+@mixin stamp-duty__col {
+  display: inline;
+  float: left;
+  width: 100%;
+
+  @include respond-to($mortgagecalc_mq-m) {
+    width: 50%;
+  }
+}
+
```

Comment:
Hi @johnplayer1982, unfortunately the margin that comes with the column mixin upsets the alignment. 

![screen shot 2016-12-01 at 10 37 40](https://cloud.githubusercontent.com/assets/67151/20790943/97143810-b7b2-11e6-974c-babb72cf7d99.png)